Maquetado/scss/details.scss

```diff
@@ -125,3 +146,59 @@
   line-height: #{$line-height-base -2};
   margin-top: 5px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .book {
+    align-items: center;
+    flex-direction: column;
+    height: 100%;
+    max-height: 700px;
+    max-width: 350px;
+    width: 100%;
```

Comment:
Delete this

Reply:
He can delete this `width: 100%;` because he's using flex?